tests/JsonReaderTests.cs

```diff
@@ -766,6 +766,56 @@ public void String_AsEnum_Doesnt_Move_Reader()
         TestMovesReaderPastReadValue(JsonReader.String().AsEnum<JsonValueKind>(), /*lang=json*/ @"""Null""");
     }
 
+    [Theory]
+    [InlineData("foobar", /*lang=json*/ @"""foobar""")]
+    [InlineData(null, /*lang=json*/ "null")]
+    public void String_Nullable_With_Valid_Input(string? expected, string json)
+    {
+        var reader = JsonReader.String().Nullable(null);
```

Comment:
Based on the [earlier suggestion to default the argument](https://github.com/atifaziz/Jacob/pull/16/files#r1034969094), this and other calls can remove `null`.